motr/io_nw_xfer.c

```diff
@@ -939,6 +1031,44 @@ static int target_ioreq_iofops_prepare(struct target_ioreq *ti,
 		    ioo->ioo_flags & M0_OOF_NOHOLE)
 			rw_fop->crw_flags |= M0_IO_FLAG_NOHOLE;
 
+		/* Assign the checksum buffer for traget */
+		if(filter == PA_DATA) {
+			if(m0_is_write_fop(&iofop->if_fop))	{				
+				M0_ASSERT(fop_cksm_nob != 0);
+				/* RPC layer to free crw_di_data_cksum */	
+				if( m0_buf_alloc(&rw_fop->crw_di_data_cksum, fop_cksm_nob) != 0 )
+					goto fini_fop;
+				
+				memcpy( rw_fop->crw_di_data_cksum.b_addr, 
+						ti->ti_attrbuf.b_addr + dispatched_cksm_nob,
+						fop_cksm_nob );			
+				dispatched_cksm_nob += fop_cksm_nob;
+				M0_ASSERT(dispatched_cksm_nob <= ti->ti_cksum_copied);
+				rw_fop->crw_is_data_fop = 1;
+		}
+		else {
```

Comment:
Please fix the styling of if-else blocks, here and elsewhere.

if() {
...
} else {
...
}